resources/asciidoctor/lib/docbook_compat/convert_lists.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module DocbookCompat
```

Comment:
Rubocop has started warning me that I'm bumping into classes that are "too big" so I'm starting to move groups of conversions to modules that I am mixing in to the converter. The class is still big and the tests are still big but at least the implementations are in "themed" files.